packages/js/api-core-tests/package.json

```diff
@@ -49,5 +49,18 @@
 		"*.(t|j)s?(x)": [
 			"eslint --fix"
 		]
+	},
+	"config": {
+		"ci": {
+			"lint": {
+				"command": "lint",
+				"changes": [
+					"data/**/*.js",
+					"endpoints/**/*.js",
+					"tests/**/*.js",
+					"utils/**/*.js"
```

Comment:
Currently these directories only contain `.js` files, but the lint command itself contains the args `--ext=js,ts,tsx`, should we be more inclusive here?

Reply:
Good catch, I must have decided to check the `--ext` option later on in the process of converting these.